server/public/styles.css

```diff
@@ -530,6 +538,10 @@ li {
   .reviewsContainer {
     width: 80%;
   }
+
+  .appDrawer {
+    width: 35%;
+  }
```

Comment:
I don't recommend a percentage width. See if you can find a good `px` value. You might not even need any break points.